### PR TITLE
Resolve #146 - 7.D Added new clause to require all Judicials to be recorded

### LINKED ISSUE #146: 7.D Create a clause that will require public Judicials to be recorded as case law
Check one:
- [X] Semantic Issue: something about the meaning of the text is different
- [ ] Non-semantic Issue: Spelling, grammar, or formatting changes.

Summary of issue:
Create a new 7.D that will require an Eboard member to record all public Judicials in a public documents accessible by all CSHers, and record all private Judicials in a private document only accessible to past, present, and future Eboard members.

### PR DESCRIPTION
**Pull request by IgorPolotai**
Check one:
- [X] Semantic Change: something about the meaning of the text is different
- [ ] Non-semantic Change: Spelling, grammar, or formatting changes.

Summary of change(s): Added a new 7.D Judicial Record Clause, that requires all Judicials to record the context, vote split, outcome, and optionally reasoning of Eboard members of all Judicials, and gives Eboard the ability to instead record the Judicial in a private document with a Simple Majority Vote (however, the content requirements are the same).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,5 @@
 *.synctex.gz
 *.py
 *.toc
+.vs/VSWorkspaceState.json
+.vs/Constition Repo/v17/.wsuo
```
Comment (mxmeinhold) [inline .gitignore:8]: These should probably be added to your personal ignore files instead

File: constitution.tex [hunk 2]
```diff
@@ -980,6 +980,8 @@
 \asection{Judicial Ruling}
 The Judicial Board may present an official ruling following the Judicial Investigation.
 This ruling may be appealed in the same manner as an Executive Board decision \ref{Appeals}.
+\asection{Judicial Records}
+All Judicials must be recorded in a public document accessible to all Active Members and Alumni by an Executive Board member. Judicial Records must include the context, vote split, and outcome, and optionally the reasoning of why the Executive Board decided on the outcome. Executive Board can choose, through a Simple Majority Vote, to instead move the Judicial Record to a private document accessible only to past, present, and future Executive Board members, but the content requirements do not change.
```
Comment (mxmeinhold) [inline constitution.tex:984]: Format: one sentance per line please

**Review by mxmeinhold — CHANGES_REQUESTED** [review]

**Comment by esoccoli** [comment]
I think the content is fine, but the wording should be changed. I would suggest changing the wording to "The Executive Board must record records of all Judicials in a location that all members can access. These records must include the context, vote split, and outcome. If it is deemed necessary, the Executive Board may choose, through a Simple Majority Vote, to move the Judicial Record to a private document that is only accessible by the Executive Board"

File: constitution.tex (hunk 2 above)
Comment (esoccoli) [inline constitution.tex:984]: I think the content is fine, but the wording should be changed. I would suggest changing the wording to "The Executive Board must record records of all Judicials in a location that all members can access. These records must include the context, vote split, and outcome. If it is deemed necessary, the Executive Board may choose, through a Simple Majority Vote, to move the Judicial Record to a private document that is only accessible by the Executive Board"

**Comment by esoccoli** [comment]
This PR contains changes made in a different PR. Make sure to rebase so there is only 1 change per PR

**Comment by esoccoli** [comment]
This will be handled in its own amendment, after we finish the rewrite

**Comment by Sam-Cordry** [comment]
Closing per discussion in meeting. Will come back to this as its own amendment at a later time.

**Pull request closed without merging**